frontend/tsconfig.json

```diff
@@ -1,6 +1,14 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
-    "strict": true
-  }
+    "strict": true,
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"],
+      "@/assets/*": ["assets/*"]
+    },
+    "allowImportingTsExtensions": true,
```

Comment:
chỗ này e pull về đang bị error nma có thể là do typescript version của em

Reply:
Error này hay như thế nào em [https://stackoverflow.com/questions/76241919/how-can-i-fix-error-ts5023-throwed-by-adding-allowimportingtsextensions-in-tsc](url)